docs/tutorial/notebook/01_config_files.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<p>See Readme.md for how to import Config</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>